### PR TITLE
Designer

### PR DESCRIPTION
**Pull request by jsvdvis**
Known bugs:
-API Variable mapping that uses variable from parent, if a rule change causes the step to lose its parent this causes a failure
-Loading a Sequential model causes a 500 (internal server error), since these models have a different structure. I think I know how to fix this, I just haven't had the time yet.
-Changing the title of a step does not yet change it in the target section of the rules. Same thing as above, haven't had the time to fix it yet.
-Result-representations cannot be saved/loaded yet.

### CODE REVIEW

File: resources/views/graph.blade.php [hunk 1]
```diff
@@ -41,6 +42,50 @@
...
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+        <th width="5%"></th>
+      </tr>
+      <tr>
+        <?php
```
Comment (Dansuf) [inline resources/views/graph.blade.php]: try to use blade template tag @for or convert <?php to @php if the php block is really unavoidable

File: resources/views/graph.blade.php [hunk 2]
```diff
@@ -41,6 +42,50 @@
...
+          <?php
+            }
+          }
+          echo "</tr>";
+        }
+        ?>
+      </tr></table>
+      <br />
+      <br />
+      <h5>Model results show that <kbd><?php echo $result/10?></kbd> in 10 patients with similar conditions like yours tested positive for <?php echo $decodeRes['title'] ?> </h5>
```
Comment (Dansuf) [inline resources/views/graph.blade.php]: use {{ $result/10 }}, the same in the following lines

File: resources/views/partials/sidebar.blade.php [hunk 3]
```diff
@@ -1,15 +1,12 @@
...
     <nav id="sidebar">
         <div id="dismiss">
-            <i class="fas fa-angle-left"></i>
+            <i class="fo-icon icon-left-open">&#xe801;</i>
         </div>
 
         <div class="sidebar-header">
             <h3>
-                {{ Auth::user()->name }}
+                    {{ Auth::user()->name }}
```
Comment (Dansuf) [inline resources/views/partials/sidebar.blade.php]: change to  {{ $user->first_name }} {{ $user->last_name }}
(yeah, I know this is my fault)

File: resources/views/workflow.blade.php [hunk 4]
```diff
@@ -3,35 +3,31 @@
...
 {{--makes inputs for all the required variables--}}
-<?php if (!empty($decodeRes)): ?>
+
+@if (!empty($result))
 <div class="container">
-  <h3><?php echo $decodeRes['title'] ?></h3>
+  <h3><?php echo $result['title'] ?></h3>
 </div>
 <div class="container">
+  <h5><?php echo $result['steps'][0]['title'] ?></h5>
```
Comment (Dansuf) [inline resources/views/workflow.blade.php]: {{ $result['steps'][0]['title'] }}

**Review by Dansuf — COMMENTED** [review]
I'll accept and merge the code as I mostly encountered code style-related errors and we need to have the branch ready for the hallway testing. However, I expect my remarks to be addressed. (I don't recommend making a review at 1am)

File: resources/assets/fontello/LICENSE.txt [hunk 5]
```diff
@@ -0,0 +1,12 @@
+Font license info
```
[the remark this answers is not in the document]
Comment (jsvdvis) [reply resources/assets/fontello/LICENSE.txt:1]: Yes, but then you get the whole fontm which can be relatively large.

File: resources/views/designer.blade.php [hunk 6]
```diff
@@ -1,66 +1,49 @@
-@extends('layouts.app')
+@extends('layouts.app') @section('content') @include('partials.sidebar')
```
[the remark this answers is not in the document]
Comment (jsvdvis) [reply resources/views/designer.blade.php:1]: Yes, not sure when this happened. Changed.

File: resources/views/partials/sidebar.blade.php (hunk 3 above)
Comment (jsvdvis) [reply resources/views/partials/sidebar.blade.php:9]: Apart from that we cannot use `$user`, since this variable is not set, I've changed it.

**Pull request merged**